client/src/App.js

```diff
@@ -1,45 +1,39 @@
-import React from 'react';
+import React, {Component} from 'react';
 import SelectedFoods from './SelectedFoods';
 import FoodSearch from './FoodSearch';
 
-const App = React.createClass({
-  getInitialState: function () {
-    return {
-      selectedFoods: [],
-    };
-  },
-  render: function () {
+class App extends Component {
+  state = {
```

Comment:
Yeah, the Class Properties syntax is indeed currently a Stage 2 proposal.  However, I double-checked the `package.json` contents before I rewrote the file, and the project currently has Stage 0 enabled.  So, the Class Properties syntax should work just fine.

Also, Class Properties are the React team's preferred approach to handle method binding (per Dan Abramov's comments at https://www.reddit.com/r/reactjs/comments/54xnao/fat_arrow_vs_autobind_vs_bindbindbindbindbind/d85wj0l/ ).  

If there's a preference for _not_ using Class Properties in this repo, the example can definitely be updated to follow suit, but given that the project has them enabled I figured it would make sense to use them.